NuRadioMC/simulation/simulation.py

```diff
@@ -139,6 +139,9 @@ def __init__(self, inputfilename,
                 new_cfg = merge_config(local_config, self._cfg)
                 self._cfg = new_cfg
 
+        if(self._cfg['seed'] is None):
```

Comment:
Maybe add a comment describing what you are doing here? This could easily confuse someone

Reply:
done